amqp/method_framing.py

```diff
@@ -157,6 +157,10 @@ def frame_writer(connection, transport,
                 offset += 8 + framelen
 
                 framelen = len(body)
+
+                if isinstance(body, str):
+                    body = body.encode()
+
```

Comment:
I hate Python 3 for requiring me to unnecessarily copy this string!   There should be a way for the new bytes object to use the strings underlying C array